tool/docker/singa/.bashrc

```diff
@@ -0,0 +1,8 @@
+export LIBRARY_PATH=/opt/OpenBLAS/lib:$LIBRARY_PATH
```

Comment:
will the .bashrc be the same as that under ~/, i.e., it will be executed by shell automatically once we login into a docker node?
this file is similar to that under tool/docker/mesos.
If docker/mesos is installed after docker/singa, then we can remove that file.